hammer_time/hammer_time.py

```diff
@@ -140,6 +141,34 @@ def perform(cls, client, machine_id):
         ctrl_client.juju('ssh', (machine_id,) + cls.kill_script)
 
 
+class InterruptNetworkAction(MachineAction):
+
+    def get_command():
+        deny_all = '; '.join([
+            'iptables --flush',
+            'iptables -P FORWARD DROP',
+            'iptables -P OUTPUT DROP',
+            'iptables -P INPUT DROP',
+            ])
+        restore = 'iptables-restore < $HOME/iptables'
+        commands = [
+            'set -eux',
+            'sudo iptables-save > $HOME/iptables',
+            'echo "{}"| sudo at now + 6 minutes'.format(restore),
+            'echo "{}"| sudo at now + 1 minutes'.format(deny_all),
+            ]
+        return '; '.join(commands)
```

Comment:
Shite interface wont let me approve. I cut off my message. Approved if I cannot find the approve button.